src/dags/importscripts/import_rdw.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+
+def download_data(endpoint: str, output_file: str) -> None:
+    """Retrieves data from source RDW
+
+    Note:
+    The RDW source has several options to retreive data (API, download).
+    Since the resources needed for the RDW dataset contain a lot of data,
+    going up to a couple of GB's per resource, the RDW resources are collected
+    by means of a csv download.
+    Getting the data by csv is by far the fastest way to retreive the data, then
+    for instance getting the data by means of several API calls.
+
+    Args:
+        endpoint: the URL specification for the data service to call
+        output_file: Name of file to save data
+
+    Result:
+        Stores the data in csv to a file.
+
+    """
+    with open(output_file, "w+") as f:
+        try:
+            request = requests.get(endpoint)
+            request.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise SystemExit(e) from e
```

Comment:
❌  You completely shut down the airflow process when the download fails... 😵  This looks like it's copy-pasted from a CLI program.

Reply:
If you want to fail a task without having Airflow retry it, you should raise an [AirflowFailException](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/exceptions/index.html?highlight=error%20handling#airflow.exceptions.AirflowFailException)